commit-message.md

```diff
@@ -0,0 +1,36 @@
+# Guidelines to create great commit messages
+
+
+We use [GitCop](https://gitcop.com) to check that commit messages are
+properly written. The rules are the following:
+
+* The first line of a commit message, called the subject line should
+  not be more than 80 characters long.
+
+* The commit message should end with the following trailers:
+
+  ```
+  Licence: MIT
+  Signed-off-by: User Name <email@address>
+  ```
+
+  where "User Name" is the author's real name and email@address one of
```

Comment:
@wking Contribution signing (any of commit/tag/pr) is much preferred even in the non-pseudonymous case. It just happens that it is still not practical nowadays (for large scale projects). When it becomes practical it will enable pseudonymous contributor to verify itself. From your viewpoint, I wonder if this is an unnecessary side effect, or should be prevented by any means? For the latter, e.g. every key has to be tied to a gov id.